server/src/logic/services/user.service.ts

```diff
@@ -1,5 +1,6 @@
 import { IDatabase } from "pg-promise";
```

Comment:
How do you get this error? If I try to register twice with the same email I got the error AUTH_USED_PASSWORD_EMAIL